src/components/map/Map.tsx

```diff
@@ -317,11 +441,19 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     width: normalize(60)
   },
+  hitbox: {
+    alignItems: 'center',
+    height: normalize(HITBOX_SIZE),
+    justifyContent: 'center',
+    width: normalize(HITBOX_SIZE)
+  },
   container: {
     alignItems: 'center',
     backgroundColor: colors.surface,
     flex: 1,
-    justifyContent: 'center'
+    height: normalize(200),
```

Comment:
why 200 here and 250 in settings? how come the difference? shouldn't we have small maps always the same height?

Reply:
usually yes, but if you check the badbelzig app there is also a difference in the set alternative location screen and the small map screen, therefore I chose those two sizes for the same design.

Reply:
sagsunseinfach is not a topic for the main code base. so please align them.

Reply:
okay, will be set to 200 for both screens in the upcoming commit.